### PR TITLE
zh-cn(translate): /web/api webvtt

### PR DESCRIPTION
**Pull request by fwqaaq**

### CODE REVIEW

**Comment by Jack-Zhang-1314** [comment]
* This time I used `ffmpeg` to test it very seriously, this time the quality should be high.
* But there may still be some issues.

File: files/zh-cn/web/api/webvtt_api/index.md [hunk 1]
```diff
@@ -159,11 +157,11 @@ A comment consists of three parts:
   - Ta en kopp
   ```
 
-## Styling WebVTT cues
+## WebVTT cue 的样式
```
Comment (yin1999) [inline files/zh-cn/web/api/webvtt_api/index.md:160]:
```suggestion
## 为 WebVTT cue 添加样式
```

File: files/zh-cn/web/api/webvtt_api/index.md [hunk 2]
```diff
@@ -159,11 +157,11 @@ A comment consists of three parts:
...
 
-You can style WebVTT cues by looking for elements which match the {{cssxref("::cue")}} pseudo-element.
+你可以通过查找与 {{cssxref("::cue")}} 伪元素匹配的元素来设计 WebVTT 样式。
```
Comment (yin1999) [inline files/zh-cn/web/api/webvtt_api/index.md:162]: 一般，我们会说“添加样式”：

```suggestion
你可以通过查找与 {{cssxref("::cue")}} 伪元素匹配的元素来为 WebVTT 添加样式。
```

**Review by yin1999 — APPROVED** [review]
Thanks @Jack-Zhang-1314, this one is much better, I have some suggestions for `styling`

**Comment by Jack-Zhang-1314** [comment]
Thank you for your continued help.

**Pull request merged**